cypress/integration/26-register_publication/275-creator_navigates_to_files_and_license_tab/files_and_license.js

```diff
@@ -6,14 +6,8 @@ import 'cypress-file-upload';
 const testFile = 'example.txt';
 
 Given('Creator begins registering a Registration in the Wizard', () => {
-  cy.loginCognito(USER_WITH_AUTHOR).then((idToken) => {
+  cy.login(USER_WITH_AUTHOR, cy.startRegistrationWithFile, testFile).then((idToken) => {
     cy.wrap(idToken).as('idToken');
-    cy.setLocalStorage('i18nextLng', 'eng');
-    cy.setLocalStorage('previouslyLoggedIn', 'true');
-    cy.visit('/');
-    cy.get('[data-testid=new-publication]').click({ force: true });
-    cy.get('[data-testid=new-publication-file]').click({ force: true });
-    cy.get('input[type=file]').attachFile(testFile);

```

Comment:
skulle ikke `cy.startRegistrationWithFile()` heller blitt lagt hit?

Reply:
Det er egentlig det jeg lurer på at jeg kanskje gjør det for komplisert. Hvis dere syns det er mer oversiktlig så gjør jeg det